depmapomics/ccle.py

```diff
@@ -0,0 +1,1423 @@
+        minsimi (float, optional): the minimum similarity to use for comparison to previous dataset. Defaults to 0.95.
+        dataset_description (str, optional): the taiga dataset description to use. Defaults to RNAseqreadme.
+        tocompare (dict, optional): the columns to compare. Defaults to {"genes_expected_count": "CCLE_RNAseq_reads", "genes_tpm": "CCLE_expression_full", "proteincoding_genes_tpm": "CCLE_expression"}.
+        rsemfilelocs (pd.DataFrame, optional): locations of RSEM output files if refworkspace is not provided (bypass interaction with terra)
+        samplesinset (list[str], optional): list of samples in the sampleset if refworkspace is not provided (bypass interaction with terra)
+        rnaqclocs (dict(str:list[str]), optional): dict(sample_id:list[QC_filepaths]) of rna qc file locations if refworkspace is not provided (bypass interaction with terra)
+    """
+    from taigapy import TaigaClient
+
+    tc = TaigaClient()
+
+    if prevcounts == "ccle":
+        prevcounts = tc.get(name=TAIGA_ETERNAL, file="CCLE_RNAseq_reads")
+
+    ccle_refsamples = trackerobj.read_tracker()
+    if gumbo:
+        ccle_refsamples = trackerobj.read_seq_table()
+
+    todrop += ccle_refsamples[
+        (ccle_refsamples.blacklist == 1) & (ccle_refsamples.datatype == "rna")
+    ].index.tolist()
+    priority = ccle_refsamples[
+        (ccle_refsamples.prioritized == 1) & (ccle_refsamples.datatype == "rna")
+    ].index.tolist()
+
+    folder = os.path.join("output", samplesetname, "")
+
+    if dry_run:
+        folder = os.path.join("output", "dryrun", "")
+
+    h.createFoldersFor(folder)
+    files, failed, _, renaming, lowqual, _ = await expressions.postProcess(
+        refworkspace,
+        samplesetname,
+        save_output=folder,
+        doCleanup=doCleanup,
+        priority=priority,
+        colstoclean=colstoclean,
+        ensemblserver=ensemblserver,
+        todrop=todrop,
+        samplesetToLoad=samplesetToLoad,
+        geneLevelCols=RSEMFILENAME_GENE,
+        trancriptLevelCols=RSEMFILENAME_TRANSCRIPTS,  # compute_enrichment=False,
+        ssGSEAcol="genes_tpm",
+        dropNonMatching=dropNonMatching,
+        dry_run=dry_run,
+        samplesinset=samplesinset,
+        rsemfilelocs=rsemfilelocs,
+        rnaqclocs=rnaqclocs,
+        **kwargs,
+    )
+
+    print("doing validation")
+    nonoverlap = set(prevcounts.columns) ^ set(
+        files.get("genes_expected_count").columns
+    )
+    print("number of non overlaping genes:")
+    print(len(nonoverlap))
+    # have we lost any samples compared to last release?
+    lost = set(prevcounts.index) - set(files.get("genes_expected_count").index)
+    print("of which, lost genes:")
+    print(lost)
+    # do we have samples that are missanotated compared to previous releases (replicate level)
+    # notindataset, missannotated, unmatched = findMissAnnotatedReplicates(replevel, prevcounts, renaming)
```

Comment:
why is this not run?

Reply:
Both these and the next block have been commented out since I joined, I'm not sure why either...

Reply:
this is more complicated here. we want our pipeline to work with a very minimal amount of information. Here renaming doesn't work as we don't want to suppose that there is relationship between the samples. Let's move it to the QC part too? (also I don't know what renaming contains but I think it is useless now)

also we can drop some inputs in this file, like :
    tocompare={
        "genes_expected_count": "CCLE_RNAseq_reads",
        "genes_tpm": "CCLE_expression_full",
        "proteincoding_genes_tpm": "CCLE_expression",
    },